modules/core/src/v2/coins/xlm.ts

```diff
@@ -503,6 +527,22 @@ export class Xlm extends BaseCoin {
     return true;
   }
 
+  /**
+   * Get extra parameters for prebuilding a tx
+   * Set empty recipients array in trustline txs
+   */
+  getExtraPrebuildParams(buildParams: ExtraPrebuildParamsOptions, callback?: NodeCallback<BuildOptions>): Bluebird<BuildOptions> {
+    return co(function *() {
```

Comment:
Nit: Coroutine here is just adding unneeded boilerplate, consider returning `Bluebird.resolve(params).asCallback(callback)` instead;

Reply:
done